proto/viam/app/data/v1/data.proto

```diff
@@ -59,18 +59,6 @@ enum Order {
   ORDER_ASCENDING = 2;
 }
 
-enum Status {
```

Comment:
Removed in corresponding app PR. We were otherwise always returning STATUS_SUCCESS since we changed our deletion API to not support partial deletion.